src/hevm/src/EVM/UnitTest.hs

```diff
@@ -74,6 +75,7 @@ data UnitTestOptions = UnitTestOptions
   , verbose     :: Maybe Int
   , maxIter     :: Maybe Integer
   , askSmtIters :: Maybe Integer
+  , debug       :: Bool
```

Comment:
could we call this smtdebug? just to make the distinction between dumping the smt queries and running the visual debugger clear?

Reply:
Sure! Do you think we should change the `bebug` option in `VeriOpts` as well or only this one?

Reply:
Left `VeriOpts` as is since it prints nonSMT stuff as well